benchmarking/evaluations/metrics/average_solo_status.py

```diff
@@ -0,0 +1,44 @@
+from collections import Counter
+from typing import Dict, List
+
+from api.models.team_set import TeamSet
+from benchmarking.evaluations.interfaces import TeamSetMetric
+
+
+class AverageSoloStatus(TeamSetMetric):
+    """
+    Calculate the average number of students having solo status in a team
+    """
+
+    def __init__(self, minority_groups: Dict[int, List[int]], *args, **kwargs):
```

Comment:
nit;

I think what made/makes this confusing is the lack of context/explanation on the meaning of `minority groups`